Add accessibility conformity mention (#141)
ambanum/OpenTermsArchive.org: src/translations/en/accessibility.mdx

# Accessibility Statement

Established on June 15, 2022.

The Ministry of Europe and Foreign Affairs is committed to making its service accessible, in accordance with Article 47 of Law No. 2005-102 of 11 February 2005.

This accessibility statement applies to **Open Terms Archive** (https://opentermsarchive.org).

## Compliance status

**Open Terms Archive** is **not compliant** with the RGAA. The site has not yet been audited.

## Improvement and contact

If you are unable to access a content or service, you can contact the Open Terms Archive manager to be directed to an accessible alternative or to obtain the content in another form.

> - E-mail: [contact@opentermsarchive.org](mailto:contact@opentermsarchive.orgcontact@opentermsarchive.orgcontact@opentermsarchive.org)

Reviewer: ```suggestion
- E-mail: [contact@opentermsarchive.org](mailto:contact@opentermsarchive.org)
```

- Twitter: @openTerms

We try to respond within 2 business days.

## Recourse

This procedure is to be used in the following case: you have notified the website manager of a lack of accessibility that prevents you from accessing a content or service of the portal and you have not received a satisfactory response.

You can :

- Write a message to the [Human Rights Defender](https://formulaire.defenseurdesdroits.fr/)

> - Contact [the representative of the Defender of Rights in your region](https://www.defenseurdesdroits.fr/saisir/delegues)

Reviewer: ```suggestion
- Contact [the representative of the Défenseur des droits in your region](https://www.defenseurdesdroits.fr/saisir/delegues)
```

This accessibility statement applies to **Open Terms Archive** (https://opentermsarchive.org).

## Compliance status

**Open Terms Archive** is **not compliant** with the RGAA. The site has not yet been audited.

## Improvement and contact

If you are unable to access a content or service, you can contact the Open Terms Archive manager to be directed to an accessible alternative or to obtain the content in another form.

- E-mail: [contact@opentermsarchive.org](mailto:contact@opentermsarchive.org)
- Twitter: [@OpenTerms](https://twitter.com/OpenTerms)

We try to respond within 2 business days.

## Recourse

This procedure is to be used in the following case: you have notified the website manager of a lack of accessibility that prevents you from accessing a content or service of the portal and you have not received a satisfactory response.

You can:

- Write a message to the [Défenseur des droits](https://formulaire.defenseurdesdroits.fr/)
- Contact [the representative of the Défenseur des droits in your region](https://www.defenseurdesdroits.fr/saisir/delegues)

> - Send a letter by post (free of charge, do not put a stamp):

Reviewer: ```suggestion
- Send a letter by post (free of charge in France, do not put a stamp):
```
